src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/GetCommandsForRebuildIndexes.sql

```diff
@@ -0,0 +1,154 @@
+﻿GO
+CREATE OR ALTER   PROCEDURE [dbo].[GetCommandsForRebuildIndexes]
+@RebuildClustered BIT
+WITH EXECUTE AS 'dbo'
```

Comment:
Please restore original formatting for all stored procedures

Reply:
Ok, I will restore all stored procedures.